platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -48,9 +56,14 @@ static const odp_crypto_cipher_capability_t cipher_capa_null[] = {
 static const odp_crypto_auth_capability_t auth_capa_null[] = {
 {.digest_len = 0, .key_len = 0, .aad_len = {.min = 0, .max = 0, .inc = 0} } };
 
+static const odp_crypto_auth_capability_t auth_capa_aes_gcm[] = {
+{.digest_len = 16, .key_len = 0, .aad_len = {.min = 8, .max = 12, .inc = 4} } };
+
 /** Forward declaration of session structure */
 typedef struct odp_crypto_generic_session_t odp_crypto_generic_session_t;
 
+static uint8_t ZEROS[2] = { 0, 0};
```

Comment:
The length of the array is not sufficient. The later copying code will read past the end of the array. Max digest length is 16 bytes, not 16 bits.

This identifier should not be capitalized.

Actually, this is not needed at all (see other comments).